.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,24 @@
-###### Before submitting the PR, please make sure you do the following
+Closes https://github.com/metabase/metabase/pull/[issue number]
 
-- [ ] If you're attempting to fix a translation issue, please submit your changes to our [POEditor project](https://poeditor.com/join/project/ynjQmwSsGh) instead of opening a PR.
+### Description
 
-### Tests
+Describe the overall approach and the problem being solved.
+
+### How to verify
+
+Describe the steps to verify that the changes are working as expected.
+
+1. New question -> Sample Dataset -> ...
+2. ...
+
+#### Before & After Screenshots
+
+_Insert images here if sensible or remove the section_
+
+### Checklist
 
-- [ ] Run the frontend and Cypress end-to-end tests with `yarn lint && yarn test`)
-- [ ] If there are changes to the backend codebase, run the backend tests with `clojure -X:dev:test`
 - [ ] Sign the [Contributor License Agreement](https://docs.google.com/a/metabase.com/forms/d/1oV38o7b9ONFSwuzwmERRMi9SYrhYeOrkbmNaq9pOJ_E/viewform)
       (unless it's a tiny documentation change).
+- [ ] Tests for the new functionality have been added.
```

Comment:
Maybe something like

`Tests have been added/updated to cover changes in this PR`

? 

Because the PR may not be new functionality etc.